.taskcluster.yml

```diff
@@ -0,0 +1,26 @@
+version: 1
+policy:
+  pullRequests: public
+tasks:
+  $match:
+    '(tasks_for == "github-pull-request" && event["action"] in [pull_request.opened,pull_request.reopened,pull_request.synchronize])':
```

Comment:
That's what the "getting started with github" page generated, but that doesn't seem to be correct based on https://community-tc.services.mozilla.com/docs/reference/integrations/github/taskcluster-yml-v1. `pull_request` shouldn't be used here, instead it should be `in ["opened", "reopened", "synchronize"]`. It seems that the generator has some issues.